src/components/screen/SearchUser.js

```diff
@@ -29,32 +39,168 @@ const styles: Styles = StyleSheet.create({
     alignItems: 'center',
     justifyContent: 'center',
   },
+  viewSearch: {
+    width: '100%',
+    justifyContent: 'center',
+    // backgroundColor: colors.dodgerBlue,
+    overflow: 'hidden',
+  },
+  imgSearch: {
+    position: 'absolute',
+    width: 16,
+    height: 16,
+    left: 30,
+    top: 18,
+  },
 });
+const AnimatedFlatList = Animated.createAnimatedComponent(FlatList);
 
 type Props = {
-
+  navigation: any;
 };
 type State = {
-
+  users: Friend[]
 };
 
 class Screen extends Component<Props, State> {
   static navigationOptions = {
-    title: 'Title',
+    title: 'Search User',
   };
 
+  scrollY = new Animated.Value(0);
+  searchTxt: string = '';
+  users: Friend[];
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      users: [
```

Comment:
I hope you to utilize model classes here.
![image](https://user-images.githubusercontent.com/27461460/52040221-116b6700-257a-11e9-8a5c-c5131ec1c798.png)

Reply:
OK. I committed about this action!